SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Reporting Security Issues
```

Comment:
I'd prefer this to go into README.md, so that it gets rendered in the default github project view. It also creates clutter in the project root dir. It does [seem fairly common](https://github.com/search?q=SECURITY.md&type=code), though -- which tools look at this? If this is about [.well-known/security.txt](https://github.com/cockpit-project/cockpit-project.github.io/pull/734), that could just point to an anchor in README or a file in doc/.